### PR TITLE
Venue Management for Event Plugin (GSoC)

### LINKED ISSUE #2522: Implement Venue Management and Booking System for Event PlugIn (GSoC)
**Is your feature request related to a problem? Please describe.**

Yes, the current event management application lacks a comprehensive venue management feature. Without the ability to effectively manage venue information and bookings, organizations face challenges in planning and executing events.

**Describe the solution you'd like**

Basic CRUD capabilities to add, update, view, and delete venue information, including details such as name, capacity, description, and image URL.

**Additional context**
This feature will significantly enhance the functionality of our event plugin making it more attractive to organizations. It will streamline event planning, reduce administrative overhead, and improve the overall experience for event attendees. The implementation of this feature is crucial for maintaining efficient and effective event management operations.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Dante291**
**What kind of change does this PR introduce?**

This PR introduces a new feature for venue management within the event management application. It adds basic CRUD capabilities to manage venue information, including name, capacity, description, and image URL.

**Issue Number:**

Fixes #2522 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa/assets/103507835/22e75f9d-4bd8-4108-83d0-cc4a90850479

**If relevant, did you update the documentation?**

Yes

**Does this PR introduce a breaking change?**

Yes

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Dante291** [comment]
@palisadoes Sorry for closing two PRs I mistakenly created them against main branch

**Review by noman2002 — CHANGES_REQUESTED** [review]
Please fix the failing tests.

File: lib/views/after_auth_screens/events/create_event_page.dart [hunk 1]
```diff
@@ -133,6 +139,106 @@ class _CreateEventPageState extends State<CreateEventPage> {
...
+                                      ),
+                                    )
+                                  else
+                                    const Image(
+                                      image: AssetImage(
+                                        'assets/images/defaultImg.png',
+                                      ),
+                                      width: 50,
+                                      height: 50,
+                                      fit: BoxFit.cover,
```
Comment (noman2002) [inline lib/views/after_auth_screens/events/create_event_page.dart:199]: Use Image.asset everywhere instead of Image(image:AssetImage)

**Review by noman2002 — CHANGES_REQUESTED** [review]

**Comment by palisadoes** [comment]
@noman2002 PTAL

**Review by noman2002 — APPROVED** [review]
LGTM

**Pull request merged**